eng/scripts/Update-Artifacts-List-For-Patch-Release.ps1

```diff
@@ -0,0 +1,117 @@
+param(
+  # $(Build.SourcesDirectory) - root of the repository
+  [Parameter(Mandatory=$true)][string]$SourcesDirectory,
+  # The yml file whose artifacts and additionalModules lists will be updated
+  [Parameter(Mandatory=$true)][string]$YmlToUpdate,
+  # The project list to be built for patch release
+  [Parameter(Mandatory=$true)][array] $ProjectList
+)
+
+$StartTime = $(get-date)
+. "${PSScriptRoot}/../common/scripts/common.ps1"
+
+# Verify that the yml file to update exists
+# !Test-Path doesn't work to negate as powershell thinks that !Test-Path is a cmdlet name
+if (!(Test-Path $YmlToUpdate -PathType Leaf)) {
+    LogError("$YmlToUpdate is not a valid file.")
+    exit 1
+}
+
+# Verify that the SourcesDirectory exists and is a directory
+if (!(Test-Path $SourcesDirectory -PathType Container)) {
+    LogError("$SourcesDirectory is either not a directory or does not exist.")
+    exit 1
+}
+
+if (Get-Module -ListAvailable -Name powershell-yaml) {
+    Write-Host "powershell-yml already installed"
+} else {
+    Install-Module -Name powershell-yaml -RequiredVersion 0.4.1 -Force -Scope CurrentUser
```

Comment:
You can use the PSModule-Helpers here, e.g. https://github.com/Azure/azure-sdk-for-java/blob/main/eng/common/scripts/Update-DocsMsToc.ps1#L42-L44

Reply:
Updated locally, it'll be updated with the next commit.